docs/Configuration.md

```diff
@@ -470,6 +474,175 @@ Example configuration with example log can be found in [/examples/logs_json/](/e
 [filelogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.47.0/receiver/filelogreceiver
 [opentelemetry-log-collection]: https://github.com/open-telemetry/opentelemetry-log-collection
 
+## Setting source category
+
+For many Sumo Logic customers the [source category][source_category_docs] is a crucial piece of metadata
+that allows to differentiate data coming from different sources.
+This section describes how to configure the Sumo Logic Distribution of OpenTelemetry
+to decorate data with this attribute.
+
+To decorate all the data from the collector with the same source category,
+set the `source_category` property on the [Sumo Logic exporter][sumologicexporter_docs] like this:
+
+```yaml
+exporters:
+  sumologic:
+    source_category: my/source/category
+```
+
+You can also use Sumo Logic exporter's [source templates][sumologicexporter_source_templates]
+to use other attributes' values to compose source category, like this:
+
+```yaml
+exporters:
+  sumologic:
+    source_category: "%{cloud.provider}/%{cloud.region}/%{cloud.platform}"
+```
+
+If you want data from different sources to have different source categories,
+you'll need to set a resource attribute named `_sourceCategory` earlier in the pipeline.
+See below for examples on how to do this in various scenarios.
+
+[source_category_docs]: https://help.sumologic.com/03Send-Data/Sources/04Reference-Information-for-Sources/Metadata-Naming-Conventions#source-categories
+[sumologicexporter_source_templates]: https://github.com/SumoLogic/sumologic-otel-collector/blob/main/pkg/exporter/sumologicexporter/README.md#source-templates
+
+### Setting source category on logs from files
+
+The [Filelog receiver][filelogreceiver_readme] allows to set attributes on received data.
+Here's an example on how to attach different source categories to logs from different files
+with the Filelog receiver:
+
+```yaml
```

Comment:
Perhaps add a `NOTE` section that this relies on the above mentioned exporter config?

Reply:
The Filelog receiver config does not rely on the above mentioned exporter config.